app/validators/questionnaire_schema.py

```diff
@@ -335,13 +348,22 @@ def get_list_collector_answer_ids(self, block_id):
     @lru_cache
     def get_all_answer_ids(self, block_id):
         questions = self.get_all_questions_for_block(self.blocks_by_id[block_id])
-        return {
-            answer["id"] for question in questions for answer in question["answers"]
-        }
+        answer_ids = []
```

Comment:
Since this is converted to a set before returning and only added to, couldn't this be initialised as a set, avoiding the conversion at the end?

Reply:
For some reason I thought since sets are immutable it is better to add to a list and then convert to set. Removed now, thanks!